Makefile

```diff
@@ -166,7 +166,12 @@ build_tfhe_full: install_rs_build_toolchain
 .PHONY: build_c_api # Build the C API for boolean and shortint
 build_c_api: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) build --release \
-		--features=$(TARGET_ARCH_FEATURE),boolean-c-api,shortint-c-api -p tfhe --all-targets
+		--features=$(TARGET_ARCH_FEATURE),shortint,integer,internal-keycache,boolean,__c_api,boolean-c-api,shortint-c-api \
+		-p tfhe \
+		--all-targets
+	RUSTUP_TOOLCHAIN=nightly cbindgen -c ./tfhe/cbindgen.toml ./tfhe/ -o target/release/tfhe.h
```

Comment:
use the CARGO_RS_CHECK_TOOLCHAIN here as it's guaranteed to be installed

Reply:
In reality, It only works with the nightly toolchain since it runs `cargo rustc -Zunpretty=expanded`

error : `the option ``Z`` is only accepted on the nightly compiler`

Reply:
yes sorry, just wanted to say we have a select check toolchain

Reply:
ok yes I need to use `RS_CHECK_TOOLCHAIN` not `RS_BUILD_TOOLCHAIN`